modules/dag-l1/src/main/scala/org/tessellation/dag/l1/http/p2p/L0DAGClusterClient.scala

```diff
@@ -0,0 +1,29 @@
+package org.tessellation.dag.l1.http.p2p
+
+import cats.effect.Async
+
+import org.tessellation.schema.peer.Peer
+import org.tessellation.sdk.http.p2p.PeerResponse
+import org.tessellation.sdk.http.p2p.PeerResponse.PeerResponse
+import org.tessellation.security.SecurityProvider
+
+import org.http4s.circe.CirceEntityCodec.circeEntityDecoder
+import org.http4s.client.Client
+import org.http4s.Method.POST
+import org.tessellation.dag.domain.block.L1Output
+
+trait L0DAGClusterClient[F[_]] {
+  def sendL1Output(output: L1Output): PeerResponse[F, Unit]
+}
+
+object L0DAGClusterClient {
+
+  def make[F[_]: Async: SecurityProvider](client: Client[F]): L0DAGClusterClient[F] =
+    new L0DAGClusterClient[F] {
+
+      def sendL1Output(output: L1Output): PeerResponse[F, Unit] =
+        PeerResponse[F, Unit]("block", POST)(client) { (req, c) =>
+          c.successful(req.withEntity(output)).void
+        }
+    }
```

Comment:
I think you can return `PeerResponse[F, Boolean]` and handle `false` later if needed (let's say to resend block or sth)

Reply:
Regarding resending. We could change the flow and mark the blocks as sent or not in the storage (similarly to how we store information about whether block is a tip). Then we could pull the blocks to send (find not sent blocks signed by our id), send them and only after success mark them as sent. Then whatever happens during sending we may just log it and pull again, the failed blocks should still be there.